testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -278,6 +283,10 @@ class MiniAppDisplayActivity : BaseActivity() {
                     AppPermission.getDeviceRequestCode(miniAppPermissionType)
                 )
             }
+
+            override fun miniAppShouldClose(alertInfo: MiniAppCloseAlertInfo) {
+                closeAlertInfo = alertInfo
```

Comment:
I wonder if it would be better for the Host App if we store the `MiniAppCloseAlertInfo` within the SDK, and then the Host App doesn't need to override any methods for this.

For example, when the Mini App sets the Close Alert info, we would store it in the SDK, and then we return it to the Host App when they call `MiniAppMessageBridge.miniAppShouldClose`. So the Host App would just do something like this:
```
 private fun checkCloseAlert() {
  val closeAlert = miniAppMessageBrdige.miniAppShouldClose
  if (closeAlert.shouldDisplay) {
    // ...
  }
}
```

@khairul-alam-licon @rleojoseph What do you think?

Reply:
Yeah, I have changed the implementation accordingly by this commit: https://github.com/rakutentech/android-miniapp/pull/447/commits/7dd88ea425dfc97a218c1595ee18c4080de2abcf, please review when you'll have available time 🙇